tools/devctr/Dockerfile

```diff
@@ -18,6 +18,29 @@ ENV RUSTUP_HOME=/usr/local/rust
 ENV PATH="$PATH:$CARGO_HOME/bin"
 ENV LC_ALL=C.UTF-8
 
+# Build and install Qemu vhost-user-blk backend
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        curl gpg gpg-agent \
+        python3-pip build-essential ninja-build libglib2.0-dev libpixman-1-dev flex bison \
+    && pip3 install meson \
+    && mkdir /tmp/qemu_build && cd /tmp/qemu_build \
+    && curl -sLO https://keys.openpgp.org/vks/v1/by-fingerprint/CEACC9E15534EBABB82D3FA03353C9CEF108B584 \
+    && curl -sLO https://download.qemu.org/qemu-8.1.1.tar.xz \
+    && curl -sLO https://download.qemu.org/qemu-8.1.1.tar.xz.sig \
+    && gpg --import CEACC9E15534EBABB82D3FA03353C9CEF108B584 \
+    && gpg --verify qemu-8.1.1.tar.xz.sig qemu-8.1.1.tar.xz \
+    && tar xf qemu-8.1.1.tar.xz && cd qemu-8.1.1 \
+    && ./configure && make -j contrib/vhost-user-blk/vhost-user-blk \
+    && cp -a ./build/contrib/vhost-user-blk/vhost-user-blk /usr/local/bin \
```

Comment:
Is it worth stripping the binary? Does it need any shared libraries?

Reply:
done

Reply:
I wonder where the rest of the 80MB comes from then hmm

Reply:
it looks like glib (a runtime dep) brings around 30M. I also updated poetry - the rest could be that?